src/Features/Core/Portable/UseThrowExpression/AbstractUseThrowExpressionDiagnosticAnalyzer.cs

```diff
@@ -62,7 +62,7 @@ protected override void InitializeWorker(AnalysisContext context)
                 s_registerOperationActionInfo.Invoke(startContext, new object[]
                 {
                     new Action<OperationAnalysisContext>(operationContext => AnalyzeOperation(operationContext, expressionTypeOpt)),
-                    ImmutableArray.Create(OperationKind.ThrowStatement)
+                    ImmutableArray.Create(OperationKind.ExpressionStatement)
```

Comment:
THere will likely be 100-1000x more expression statements than throw expressions.